public/pages/Dashboard/containers/Dashboard.js

```diff
@@ -192,26 +192,25 @@ export default class Dashboard extends Component {
     this.setState({ ...this.state, loadingMonitors: false, monitors: monitors });
   }
 
-  // TODO: exists in both Dashboard and Monitors, should be moved to redux when implemented
-  acknowledgeAlert = async () => {
-    const { selectedItems } = this.state;
-    const { httpClient, notifications, perAlertView } = this.props;
-
-    if (!selectedItems.length) return;
-
-    let selectedAlerts = perAlertView ? selectedItems : _.get(selectedItems, '0.alerts', []);
-    selectedAlerts = filterActiveAlerts(selectedAlerts);
+  acknowledgeAlerts = async (alerts) => {
+    const { httpClient, notifications } = this.props;
+    const selectedAlerts = filterActiveAlerts(alerts);
 
     const monitorAlerts = selectedAlerts.reduce((monitorAlerts, alert) => {
-      const { id, monitor_id: monitorId } = alert;
-      if (monitorAlerts[monitorId]) monitorAlerts[monitorId].push(id);
-      else monitorAlerts[monitorId] = [id];
+      const id = alert.id;
+      const monitorId = alert.workflow_id || alert.monitor_id;
+      if (monitorAlerts[monitorId]) monitorAlerts[monitorId].alerts.push(id);
+      else
+        monitorAlerts[monitorId] = {
+          alerts: [id],
+          poolType: !!alert.workflow_id ? 'workflows' : 'monitors',
```

Comment:
Non-blocking nitpick: Should we have a constant that contains an exhaustive list of the supported `poolType`s?